web-client/src/ducks/auth/reducer.ts

```diff
@@ -20,11 +28,93 @@ export default createReducer<AuthState>(
     ) => {
       state.token = payload.accessToken;
     },
-    [FIREBASE_FACEBOOK_LOGIN.COMPLETED]: (
+    [FIREBASE_FACEBOOK_LOGIN_POPUP.PENDING]: (state: AuthState) => ({
+      ...state,
```

Comment:
Here we don't need to use spread operator. The state object is a draft, since we're using immerJs 🚀 . You can just go ahead ad mutate the state props.

Reply:
okay, I wasn't aware of that. I will make the change

Reply:
I have made this change now